package.json

```diff
@@ -9,18 +9,28 @@
     "type": "git",
     "url": "https://github.com/Microsoft/napajs.git"
   },
+  "binary": {
+    "module_name": "napa-binding",
```

Comment:
the 'module-name' field is used exactly for node-pre-gyp searching for file <module-name>.node

---
In reply to: [133269664](https://github.com/Microsoft/napajs/pull/41#discussion_r133269664) [](ancestors = 133269664)